framework/Optimizers/gradients/SPSA.py

```diff
@@ -54,7 +54,7 @@ def getInputSpecification(cls):
           """
     return specs
 
-  def chooseEvaluationPoints(self, opt, stepSize):
+  def chooseEvaluationPoints(self, opt, stepSize, **kwargs):
     """
       Determines new point(s) needed to evaluate gradient
       @ In, opt, dict, current opt point (normalized)
```

Comment:
docstring. Maybe it should be "constraints" and not kwargs

Reply:
fixed